srcpkgs/libva-utils/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'libva-utils'
 pkgname=libva-utils
-version=2.16.0
+version=2.17.1
 revision=1
 build_style=meson
 configure_args="-Ddrm=true -Dx11=true -Dwayland=true"
 hostmakedepends="pkg-config wayland-devel"
 makedepends="libva-devel libXfixes-devel libXext-devel
  libdrm-devel wayland-devel"
 short_desc="Video Acceleration (VA) API - Utilities"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="icp <pangolin@vivaldi.net>"
 license="MIT"
-homepage="https://01.org/linuxmedia/vaapi"
+homepage="https://github.com/intel/libva-utils"
```

Comment:
More accurate for project homepage? https://www.intel.com/content/www/us/en/developer/topic-technology/open/opensource-media-framework/overview.html

Reply:
Right, I saw that but the link was so long - felt ugly putting in the template. Just for reference, the previous one is giving "access denied".

Reply:
https://01.org/linuxmedia this one works - should I use that instead?

Reply:
Sure! That just forwards to the long/actual URL, which is why I think it was (similar to) that before.